arcgis-ios-sdk-samples/Layers/RGB renderer/README.metadata.json

```diff
@@ -37,7 +37,7 @@
     ],
     "snippets": [
         "RGB Renderer Cells.swift",
-        "RGBRendererSettingsVC.swift",
+        "RGBRendererSettingsViewController.swift",
         "RGBRendererViewController.swift"
```

Comment:
This file should really come first since it is the main view of the sample. Do you mind reordering these?

Reply:
Ahh good catch. In previous README and metadata updates, we ran a script to sort the filenames by alphabetical order.

It makes sense to put the most important file atop so that users see it first on the dev website.

I can remove this ordering rule from the metadata style checker. @vquach2404 what do you think?

Reply:
I agree, the first file should always be the main view and anything that comes afterward should be alphabetically ordered.

Reply:
I personally don't think the files should be sorted automatically and I regret that they were. I somehow missed that. The file order is dependent on each sample and is often used to show the flow through a sample. For example, the metadata file for the IWA sample [originally](https://github.com/ChrisFrenchPDX/arcgis-runtime-samples-ios/blob/8176b41c91e64bdce0a29459069ecf600da933ed/arcgis-ios-sdk-samples/Cloud%20%26%20Portal/Integrated%20Windows%20Authentication/README.metadata.json#L23) had the files listed by the order in which they are used in the sample. The first presented view controller was first, the second listed second, and so on.

Reply:
Hm, we could always sort through all the samples with multiple swift files and revert the snippets portion. It may be a lower priority issue but we can make it a rule to order the list according to order of use from now on.

Reply:
Sounds good to me!